README.md

```diff
@@ -35,6 +35,10 @@ You can customize your installation by creating and editing `dist/theme/config.t
 The `dist/theme/` directory also contains files make it easy to set a different
 logo image, colors, welcome and help text.
 
+## Nominatim Docker
+
+You have to set the permissions for the webserver group(default www-data) at /nominatim/website to 755.
```

Comment:
Yes, seems outdated. The standard nominatim-docker installation doesn't use `www-data`. I added working instructions in https://github.com/osm-search/nominatim-ui/pull/233

Reply:
@lonvia 
I had to set the permissions for the folder because there is just in the guide of the vanilla Nominatim setup where the permissions for the Folder /nominatim/website, but in the setup of the Docker version the folder gets no Permissions so the User which uses the Nominatim UI services at the system cant access the folger /nominatim/website.
I discovered this and thoght that it would be quicker for others to see where a problem could be.
Maybe its just not a big issue or people who work theyr whole life with linux, but i thought it would be nice to know for those like me who arent this big into linux.

Reply:
@mtmail 
which user is then used if you use the Nominatim UI in the Docker setup?

Reply:
Afaik the nominatim-docker project uses user 'nominatim' and group 'nominatim' for their setup.

Reply:
@mtmail  i mentioned above why i suggested the addition to the docu, because i just thought it would be nice to add this info for the not well into linux educated users like me, bacause i felt this was somewhere missing and it was demanded not to add this part in the Nominatim Docker readme.